content/scripts/build.js

```diff
@@ -113,18 +119,15 @@ function buildMDNUrl(locale, slug) {
   return `/${locale}/docs/${slug}`;
 }
 
-/** Throw an error if the locale in insane.
- * This helps breaking the build if someone has put in faulty data into
- * the content (metadata file).
- * If all is well, do nothing. Nothing is expected to return.
- */
-function validateLocale(locale) {
-  if (!locale) {
-    throw new Error("locale is empty");
-  }
-  if (!VALID_LOCALES.has(locale)) {
-    throw new Error(`'${locale}' is not a recognized locale`);
+function extractLocale(filepath) {
+  let locale = undefined;
+  for (const segment of filepath.split(path.sep)) {
+    locale = VALID_LOCALES.get(segment.toLowerCase());
+    if (locale) {
+      break;
+    }
   }
+  return locale;
```

Comment:
Would you mind reviewing https://github.com/mdn/stumptown-renderer/pull/406 and once that's into `create-content-from-mysql` we can carry on with the KS rendering stuff.